acceptance/suites/pre_suite/foss/00_setup_environment.rb

```diff
@@ -1,2 +1,13 @@
-step "Initialize Test Config"
+step "Initialize Test Config" do
   PuppetServerExtensions.initialize_config options
+
+  if PuppetServerExtensions.config[:puppet_build_version] == 'LATEST'
```

Comment:
I think we should move our default behavior to "latest" but that it would be good to leave the code in place which sets the different variables in the helper.rb file as a "template" for what we need to set in the event that we need to pin to a specific version temporarily.  We may need to temporarily pin in the future if we have a prolonged period of puppet-agent failures on latest but need to release puppet-server based on whatever the last puppet-agent release version was, for example.